receipt_label/receipt_label/utils/chroma_s3_helpers.py

```
@@ -24,6 +25,19 @@
+try:
+    from .chroma_hash import (
+        calculate_chromadb_hash,
+        create_hash_file_content,
+        parse_hash_file_content,
+        compare_hash_results,
+        ChromaDBHashResult
+    )
+    HASH_UTILS_AVAILABLE = True
+except ImportError:
+    HASH_UTILS_AVAILABLE = False
+    logger.warning("ChromaDB hash utilities not available")
```

Ruff (0.12.2)

Undefined name `logger`

(F821)